static/scss/answers/collapsible-filters/sdk-filter-overrides.scss

```diff
@@ -23,8 +23,9 @@
     }
   
     &-options {
-      margin-top: 4px;
-      margin-bottom: 4px;
+      margin: 0;
```

Comment:
Is the `margin: 0` necessary here? Do the options inherit margin from somewhere up the CSS chain?

Reply:
Yes, it's inheriting margin from the SDK styling for `yxt-FilterOptions-options`